cmd/clusterctl/client/cluster/mover.go

```diff
@@ -102,6 +110,100 @@ func (o *objectMover) Move(namespace string, toCluster Client, dryRun bool) erro
 	return o.move(objectGraph, proxy)
 }
 
+func (o *objectMover) Save(namespace string, directory string) error {
```

Comment:
We should have a test for this func.
Given that we already have coverage for most of the utilities used internally I would suggest to test save with a fairly simple cluster, like e.g. https://github.com/kubernetes-sigs/cluster-api/blob/88b3ee3e1ac58440bc450126922a723cbc86f7ac/cmd/clusterctl/client/cluster/mover_test.go#L169-L179
checking that the expected files are created in a temp folder.